src/tiled/propertybrowser.h

```diff
@@ -115,6 +115,7 @@ private slots:
         YProperty,
         WidthProperty,
         HeightProperty,
+        TileOffsetXYProperty,
```

Comment:
Before renaming this to `TileOffsetProperty`, please rename the existing `TileOffsetProperty` to `TilesetTileOffsetProperty`.